sdk/storage/storage-blob/MigrationGuide.md

```diff
@@ -0,0 +1,290 @@
+# Guide for migrating to `@azure/storage-blob` v12 from `azure-storage`
+
+This guide is intended to assist in the migration to `@azure/storage-blob` from the legacy `azure-storage` package. It will focus on side-by-side comparisons for similar operations between the two packages.
```

Comment:
```suggestion
This guide is intended to assist in the migration to version 12 of `@azure/storage-blob` from the legacy `azure-storage` package. It will focus on side-by-side comparisons for similar operations between the two packages.
```

Reply:
Updated